nss-cohort-36/bangazon-client-imagination-station-react
# Lr user completes order

# Description
User should be able to complete and order by adding a payment type and should be taken to the thank you screen afterwards.
## Type of change
Please delete options that are not relevant.

# Testing Instructions
1. `git fetch --all`
2. `git checkout lr-user-completes-order `
3. Add a product to your open order, then go to orders and push complete an order, add a payment type and then you should be taken to the thank you screen afterwards.

# Checklist:
